packages/svelte2tsx/src/svelte2tsx.ts

```diff
@@ -83,6 +83,41 @@ function processSvelteTemplate(str: MagicString): TemplateProcessResult {
             str.appendLeft(parent.end, ')');
             return;
         }
+        // handle $store +=, -=, *=, /=, %=, **=
+        const operators = ['+=', '-=', '*=', '/=', '%=', '**='];
+        if (
+            parent.type == 'AssignmentExpression' &&
+            parent.left == node &&
+            operators.includes(parent.operator)
+        ) {
+            const storename = node.name.slice(1); // drop the $
+            const operator = parent.operator.substring(0, parent.operator.length - 1); // drop the = sign
+            str.overwrite(
+                parent.start,
+                str.original.indexOf('=', node.end) + 1,
+                `${storename}.set( __sveltets_store_get(${storename}) ${operator}`,
+            );
+            str.appendLeft(parent.end, ')');
+            return;
+        }
+        // handle $store++, $store--, ++$store, --$store
+        // https://github.com/sveltejs/language-tools/issues/170
+        if (parent.type == 'UpdateExpression') {
+            let appendor;
+            if (parent.operator === '++') appendor = '+';
+            if (parent.operator === '--') appendor = '-';
+            if (appendor === undefined)
+                throw new Error(
+                    'warning - unrecognized UpdateExpression operator! Probably some edge case unaccounted for in svelte2tsx, please file an issue.',
+                );
```

Comment:
not actually sure if this is needed but just being cautious - are there any other operators like ++ and --? i dont know.